.eslintrc

```diff
@@ -0,0 +1,50 @@
+{
+    "env": {
+        "browser": true,
+        "commonjs": true,
+        "es6": false,
+        "node": false
+    },
+    "parserOptions": {
+        "ecmaFeatures": {
+            "jsx": true
+        },
+        "ecmaVersion": 5,
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react"
+    ],
+    "rules": {
+        "no-unreachable": ["error"],
+        "no-continue": ["error"],
+        "no-unused-vars": ["warn"],
+        "no-param-reassign": ["warn"],
+        "valid-typeof": ["warn"],
+        "space-before-blocks": ["warn"],
+        "keyword-spacing": ["warn"],
+        "spaced-comment": ["warn"],
+        "brace-style": ["warn"],
+        "one-var-declaration-per-line": ["warn"],
+        "no-magic-numbers": ["warn", { "ignore": [0, -1, 1] }],
+        "valid-jsdoc": ["warn"],
+        "camelcase": ["warn"],
+
```

Comment:
Is there a reason for these extra empty lines? Just out of curiosity. It doesn't immediately look like any groupings of any sort.